LEGO1/infocenterstate.h

```diff
@@ -23,6 +23,38 @@ class InfocenterState : public LegoState
   {
     return !strcmp(name, InfocenterState::ClassName()) || LegoState::IsA(name);
   }
+
+  inline MxU32 GetSomething(int p_index) { return m_buffer[p_index]; }
+  inline void SetSomething(int p_index, MxU32 p_value) { m_buffer[p_index] = p_value; }
+
+private:
+  // Size: 0xC
+  struct SomeStruct
+  {
+    MxU32 unk1;
+    MxU16 unk2;
+    MxU16 unk3;
+    MxU16 unk4;
+    MxU16 padding;
+  };
+
+  MxU16 unk1;
+  MxU16 unk2;
+  MxU32 unk3;
+  MxU32 padding1;
+  void *unk4;
+  MxU16 unk5;
+  MxU16 unk6;
+  MxU16 unk7;
+  MxU16 padding2;
+  void *unk8;
+  MxU16 unk9;
+  MxU16 unk10;
+  MxU16 unk11;
+  MxU16 padding3;
```

Comment:
Unknown vars should preferably use the `undefined` typedefs in `decomp.h`. Also are the `padding` members necessary? I would have thought the compiler would fill those in automatically.

Reply:
Not only will it but it has to due to alignment rules, I just prefer to be explicit so I can mess something up while transcribing what I did in Ghidra. I'll avoid it if you don't like the clutter.